.gitlab-ci.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 ETH Zurich and University of Bologna.
```

Comment:
This file it might make sense to transition into a subpipe in another repository on our internal git (similar to carfield or cheshire), that way we can also run more complex tools (spyglass, elaboration, ...) without putting all the required scripts here. But for a start I think this is ok.